docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -215,6 +215,12 @@ It is available from [nuget.org](https://www.nuget.org/packages/Azure.DCAP.Windo
 nuget.exe install Azure.DCAP.Windows -ExcludeVersion -Version 0.0.2 -OutputDirectory C:\openenclave\prereqs\nuget
 ```
 
+NOTE: After instaling the Azure.DCAP.Windows package you will need to install the dcap_quoteprov.dll to the desired
```

Comment:
instaling -> installing

Reply:
I talked Radhika and confirmed that this extra note is not necessary.